src/acl.js

```diff
@@ -0,0 +1,53 @@
+class ACL {
+  constructor (keystore, key, keys = []) {
+    this._keystore = keystore
```

Comment:
I think here it should only get `sign`, `getPubKey` and `verify` functions instead of direct access to the keys